### DISCUSSION TITLE
SEB Black Screen

### DISCUSSION THREAD

**Discussion by bbestrella** [SafeExamBrowser/seb-win-refactoring#180] [Q&A]
Hi,

When I start SEB, it only shows black screen. I can only see the panel at the bottom of the screen. Appreciate help please. Thank you.

Here's the SebClient log:
02/09/2021 6:01:10 pm [Information]: SEB version: 2.4.7485.27501

02/09/2021 6:01:10 pm [Error]: Could not load SebClientSettigs.seb from the Program Data directorySystem.IO.FileNotFoundExceptionCould not find file 'C:\ProgramData\SafeExamBrowser\SebClientSettings.seb'.

02/09/2021 6:01:10 pm [Information]: User Name: estrellabb Host Name: localhost Port Number: 57016 Send Interval: 100 Recv Timeout: 100 Num Messages: 3 SebClientConfigFileDirectory: C:\Users\estrellabb\AppData\Roaming\SafeExamBrowser\ SebClientConfigFile: C:\Users\estrellabb\AppData\Roaming\SafeExamBrowser\SebClientSettings.seb

02/09/2021 6:01:10 pm [Information]: SEB client configuration set in InitSebSettings().

02/09/2021 6:01:10 pm [Information]: OS Version: 2050

02/09/2021 6:01:10 pm [Information]: Successfully InitSebSettings

02/09/2021 6:01:10 pm [Information]: 
All Running Processes:
--------------
taskhostw
igfxCUIService
chrome
SCNotification
IntelCpHDCPSvc
WmiPrvSE
svchost
svchost
svchost
RuntimeBroker
wininit
svchost
svchost
chrome
svchost
TmSSClient
fppsvc
Ntrtscan
CNTAoSMgr
IntelCpHeciSvc
SettingSyncHost
svchost
svchost
conhost
TMiACAgentSvc
dllhost
svchost
svchost
svchost
svchost
svchost
CSFalconContainer
EndPointClassifier
F1EUI
svchost
dllhost
uhssvc
svchost
chrome
WmiPrvSE
SearchIndexer
WmiPrvSE
svchost
dsagent
SearchFilterHost
dwm
YourPhone
conhost
svchost
smartscreen
RuntimeBroker
spoolsv
GoogleCrashHandler
ApplicationFrameHost
jhi_service
svchost
svchost
SafeExamBrowser
svchost
WmiPrvSE
svchost
svchost
conhost
svchost
kvoop
TrGUI
svchost
svchost
svchost
SynaMonApp
RuntimeBroker
explorer
svchost
RuntimeBroker
ShellExperienceHost
GoogleCrashHandler64
svchost
chrome
EPWD
kvoop
WmiPrvSE
kvoop
svchost
SecurityHealthService
chrome
OneApp.IGCC.WinService
svchost
jusched
SleepAgentService
winlogon
igfxEM
TracCAPI
Memory Compression
wepsvc
svchost
Dserui
svchost
ibtsiva
csrss
svchost
svchost
WmiPrvSE
MBAMAgent
wlanext
svchost
svchost
RuntimeBroker
AdobeARM
FileCoAuth
Calculator
svchost
chrome
PresentationFontCache
LogServer
svchost
svchost
LogServer
svchost
SystemSettings
svchost
csrss
kvoop
svchost
svchost
svchost
svchost
svchost
svchost
TMBMSRV
RuntimeBroker
kvoop
unsecapp
Flow
smss
svchost
kvoop
svchost
dllhost
MicTray64
SmartAudio3
svchost
kvoop
svchost
iVPAgent
conhost
svchost
fontdrvhost
svchost
svchost
svchost
SearchProtocolHost
CxUtilSvc
svchost
SgrmBroker
svchost
svchost
TracSrvWrapper
svchost
chrome
svchost
Registry
svchost
svchost
CxAudioSvc
svchost
svchost
svchost
svchost
sihost
svchost
CSFalconService
kvoop
fontdrvhost
lsass
svchost
RuntimeBroker
chrome
StartMenuExperienceHost
TmListen
svchost
svchost
ctfmon
OfficeClickToRun
svchost
CcmExec
PccNTMon
FEService
SecurityHealthSystray
svchost
tmwscsvc
CmRcService
svchost
services
svchost
chrome
SearchUI
SebWindowsServiceWCF
svchost
kvoop
TmCCSF
svchost
FEConsole
conhost
dllhost
kvoop
Video.UI
svchost
Microsoft.Photos
ShowMsg
armsvc
svchost
svchost
conhost
System
Idle

**Comment by dbuechel** [comment]
SEB version 2.x is no longer actively developed. Please try the latest release version: https://github.com/SafeExamBrowser/seb-win-refactoring/releases

**Comment by bbestrella** [comment]
Thank you for your reply. We're currently using this version. All of my colleagues were able to access, except me. It just shows black screen.

**Comment by dbuechel** [comment]
As said, version 2.x is no longer actively developed, there isn't much we can do about it. We highly recommend updating to version 3.x as soon as possible.
